lightwood/helpers/general.py

```diff
@@ -120,21 +122,25 @@ def evaluate_array_accuracy(
     Scores are computed for each timestep (as determined by `timeseries_settings.nr_predictions`),
     and the final accuracy is the reciprocal of the average score through all timesteps.
     """
-
     ts_analysis = kwargs.get('ts_analysis', {})
     naive_errors = ts_analysis.get('ts_naive_mae', {})
+    wrapped_data = {
+        'data': data.reset_index(drop=True),
+        'group_info': {gcol: data[gcol].tolist()
+                       for gcol in ts_analysis['tss'].group_by} if ts_analysis['tss'].group_by else {}
+    }
+
+    if ts_analysis['tss'].group_by:
+        [true_values.pop(gby_col) for gby_col in ts_analysis['tss'].group_by]
+
+    true_values = np.array(true_values)
```

Comment:
if its always pandas true_values.values should be acceptable too but nitpick, not a real issue

Reply:
Hmh, you are right. I guess I'll keep this one for clarity (arguable, but as you said it's a nitpick)